### ISSUE TITLE
[DE] "add apple to shoppinglist" -> "add apple" added to shopping list

### ISSUE THREAD

**Issue by Progaros** [home-assistant/intents#1941]
"schreibe _apfel_ auf die einkaufsliste" -> "_schreibe apfel_ hinzugefügt"
![image](https://github.com/home-assistant/intents/assets/22076711/1d6d8a14-fa3d-4f11-a85f-492f9017145d)

line 9 might be the problem here: https://github.com/home-assistant/intents/blob/7aeb3520b951e77f83f01b7c453df25766d41d8c/sentences/de/shopping_list_HassShoppingListAddItem.yaml#L9

switching with line 8 might work https://github.com/home-assistant/intents/blob/7aeb3520b951e77f83f01b7c453df25766d41d8c/sentences/de/shopping_list_HassShoppingListAddItem.yaml#L8) 

otherwise removal of line 9 doesn't seem to be a problem for me, since it's not even a real sentence
`<item> (auf|in) <meine_liste> [setzen|schreiben|nehmen]` translates to `to [add|write|take] <item> [to|on] <my_list>`

**Comment by dontinelli** [comment]
I think one should change `[setzen|schreiben|nehmen]` to `(setzen|schreiben|nehmen)`, as - like @Progaros said - without the verb it is no sentence.

**Comment by mib1185** [comment]
yeah, that's actually an issue with the order of the sentences, fix is coming ....
